config/main.toml

```
@@ -49,9 +49,11 @@ config_version = 1
         _preset = "state_monitor.outputs.plugin"
         power_delim = 20  # delimiter between normal power and low power
         low_space = 100  # if # of GB is less than this, generate a notification
-        method = "slack"  # the method how we're going to send notifications
+        method = "google chat"  # the method how we're going to send notifications
```

While perfectly _legal_, at least to me its a little unexpected and unusual to have a space in a programmatic lookup table key like this, so you might want to consider changing it (thought it is, of course, entirely up to you).

```suggestion
        method = "gchat"  # the method how we're going to send notifications
```